airbyte-integrations/bases/debezium-v1-4-2/src/main/java/io/airbyte/integrations/debezium/internals/MSSQLConverter.java

```diff
@@ -165,6 +168,23 @@ private void registerMoney(final RelationalColumn field,
     });
   }
 
+  private void registerDecimal(final RelationalColumn field,
+                             final ConverterRegistration<SchemaBuilder> registration) {
+    registration.register(SchemaBuilder.float64(), input -> {
+      if (Objects.isNull(input)) {
+        return DebeziumConverterUtils.convertDefaultValue(field);
+      }
+
+      if (input instanceof BigDecimal) {
+        return ((BigDecimal) input).doubleValue();
```

Comment:
I dont understand why are we converting to the double value? Whats wrong with using BigDecimal as it is? Wouldnt it result in loss of precision and scale?